### ISSUE TITLE
Add playback

### ISSUE THREAD

**Issue by Ysurac** [Ysurac/FlightAirMap#229]
Use leaflet playback for all flight in 2D.
Playback should be quite easy to add for 3D world as date already exist in the CZML file.

**Comment by xyziolite** [comment]
very nice function to have.. Thanks

**Comment by Ysurac** [comment]
This will be commited for 3D view next week.
A little bit more difficult for 2D view...

**Comment by xyziolite** [comment]
Copied Thank you .

**Comment by Ysurac** [comment]
Available for 3D view in latest commit.
Too slow if many flights, need more work...

**Comment by xyziolite** [comment]
Hi , Nice to have a wonderfull flightairmap . a great improvement . How to
add Playback function as you was commited in 3D view.. Thanks

**Comment by Ysurac** [comment]
In 3D view if you have **$globalArchive** set to **TRUE** in *require/settings.php* you should see a little box in the sidebar that will display the *Playback* menu.

**Comment by xyziolite** [comment]
Thank you..

**Comment by xyziolite** [comment]
Hi , I have Aircraft Detected 0 during playback , did i miss any more
settings, Thanks

**Comment by Ysurac** [comment]
I fixed a little issue in latest commit.
Data are put in the table **spotter_archive** when **$globalArchive** is set to **TRUE**, so if you have data in this table, all should be ok.

**Comment by xyziolite** [comment]
I used latest commit , table spotter_archive still empty . thanks

**Comment by Ysurac** [comment]
Did you restart **spotter-daemon.php** after setting **$globalArchive** to **TRUE** ?
What source do you use ? ADS-B, OGN, Vatsim, IVAO,.... ?

**Comment by xyziolite** [comment]
Yes , i did restart daemon-spotter.php after setting *$globalArchive* to *TRUE
. *i also try /install/install_db.php and /scripts/update_db.php .
i'm using ADS-B . No other Source . Thanks

**Comment by Ysurac** [comment]
You should never run **install/install_db.php** it's only if there is a problem with initial install.

All should work. You should get data in table spotter_archive. No idea.

**Comment by xyziolite** [comment]
opps , noted . I will restart the daemon-spotter again . Thank you .
